baas-tests/src/test/scala/com/ing/baker/baas/recipe/ShipItemsInstance.scala

```diff
@@ -0,0 +1,20 @@
+package com.ing.baker.baas.recipe
+
+import cats.effect.{IO, Timer}
+import cats.implicits._
+import com.ing.baker.baas.recipe.CheckoutFlowEvents.ShippingConfirmed
+import com.ing.baker.baas.recipe.CheckoutFlowIngredients.ShippingOrder
+import com.ing.baker.baas.recipe.CheckoutFlowInteractions.ShipItems
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+class ShipItemsInstance(implicit timer: Timer[IO]) extends ShipItems {
+
+  override def apply(order: ShippingOrder): Future[ShippingConfirmed] = {
+      IO.sleep(500.millis)
```

Comment:
I would like to have these to emulate computation time, since these are big integration tests which should include asynchronousity to test for race conditions etc